tests/TestCase.php

```diff
@@ -93,19 +122,15 @@ protected function clearSolrIndex() {
     protected function setUp() {
         parent::setUp();
 
+        Opus_Search_Config::dropCached();
+        Opus_Search_Service::dropCached();
+
         $this->_clearTables();
         $this->clearSolrIndex();
-
-        $config = Zend_Registry::get('Zend_Config');
-        if (!is_null($config)) {
-            $this->config_backup = clone $config;
-        }
     }
 
-    protected function  tearDown() {
-        if (!is_null($this->config_backup)) {
-            Zend_Registry::set('Zend_Config', $this->config_backup);
-        }
+    protected function tearDown() {
+        $this->clearSolrIndex();
```

Comment:
I am concentrating on the framework first, before looking at the application. Running the tests right now it seems to me, that this new line is costing quite a bit of time, although I cannot say for sure how much yet. At the beginning my local configuration wasn't updated yet and I was wondering why tests broke that had nothing to do with Solr. I think leaving TestCase and therefore most tests alone and extending it in (e.g) SearchTestCase might have been the better choice. I am I missing anything?